### PR TITLE
LV522 Fixes (Weedable tiles, Invis walls, weird windows)

### PR DESCRIPTION
**Pull request by spartanbobby**
# About the pull request

This PR aims to fix bugs with LV522 that have been brought to my attention one of them being a critical bug involving a recent change to weedable tiles

APC: APCs had a problem for a while now somehow despite being invincible it can still be destroyed I'm not much of a coder so instead of trying to fix whatever's going wrong there I've replaced the invisible blockers with rock walls that'll only be seen if the APC gets deleted

Weedable Tiles: A recent PR tried to address a problem with gardener drone for LV522 but messed up regular shale tiles making them unweedable this PR should fix both the problem the OG PR tried to fix and the once weedable tiles now being weedable again

Areas: the W-Y vault was considered OOB this PR should also fix that

Other: I also moved around some sandbags and a breaching charge just to mess with people % the sandbags being all in the same place AND being near a "meta" hold spot was pretty strong

Hardy Weeds working on layer1 shale
https://i.imgur.com/CG80UJ3.png

Weeds working on dirt
https://i.imgur.com/RWRnyPE.png

Weird windows: I'm not exactly sure what's wrong here and will look into this further for now all the problem windows have been replaced with walls and I want to reach out to players to screenshot any invisible windows they encounter on LV522 or any map so I can actually figure out whats going wrong

# Explain why it's good for the game

The bugs in this PR are frustrating to players and should be fixed

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl: SpartanBobby
fix: fixes invisible walls on LV522
fix: fixes unweedable tiles on LV522
fix: fixes invisible windows on LV522 (if you spot any invisible windows send a picture of them to the SS13: CM discord #mapperchannel @spartanbobby
fix: fixes LV522 OOB area in the W-Y vault
maptweak: moves sandbags and a breaching charge on LV522
/:cl:

### CODE REVIEW

**Comment by Steelpoint** [comment]
I made my own fix to the APC issue on this PR here

https://github.com/cmss13-devs/cmss13/pull/3563

I don't know if your or my solutions are better, though my change does mean when the APC is destroyed there are no more invisible tiles blocking movement. 

I'll leave it for the maintainers to decide though I'd be fine with closing my PR if your solution is the better one.

**Review by Drulikar — APPROVED** [review]
Changes looks good to me, but this will need maptainer approval.

Should fix #3612 and #3604

**Comment by Drulikar** [comment]
Just for reference, here is after an OB, where the APC was: 
image
I can't find any spots where I cannot pass somewhere it looks like I should be able to.

**Pull request merged**
